jenkins/Machines.groovy

```diff
@@ -33,9 +33,9 @@ def pilatus = [name: 'pilatus',
 def tsa = [name: 'tsa',
            archs: [],
            buildPath: '/tmp/$USER/easybuild/build',
-           unusePath: '/apps/arolla/UES/jenkins/RH7.7/MCH-PE20.06/generic/easybuild/modules/all',
+           unusePath: '',
            modulesProduction: '',
            modulesUnuseProduction: '',
-           prefixProduction: '/apps/arolla/UES/jenkins/RH7.7/MCH-PE20.08/generic/easybuild']
+           prefixProduction: '/apps/arolla/UES/jenkins/RH7.9/MCH-PE20.08-UP01/generic/easybuild/modules/all']
```

Comment:
```suggestion
           prefixProduction: '/apps/arolla/UES/jenkins/RH7.9/MCH-PE20.08-UP01/generic/easybuild']
```

Reply:
@kraushm I think `/modules/all` should not appear in the `prefix`, since it is added by EasyBuild

Reply:
Indeed, my bad.